rm/rm-api/src/main/java/org/ow2/proactive/resourcemanager/core/properties/PAResourceManagerProperties.java

```diff
@@ -138,6 +138,9 @@ public enum PAResourceManagerProperties implements PACommonProperties {
     /** Resource Manager group file name */
     RM_GROUP_FILE("pa.rm.defaultgroupfilename", PropertyType.STRING, "config/authentication/group.cfg"),
 
+    /** Resource Manager tenant file name */
+    RM_TENANT_FILE("pa.rm.defaulttenantfilename", PropertyType.STRING, "config/authentication/tenant.cfg"),
```

Comment:
I don't understand why having config/authentication/tenant.cfg for RM and Scheduler separately?